cmd/generate_client_api_key.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"groundcover.com/pkg/api"
+	"groundcover.com/pkg/auth"
+	"groundcover.com/pkg/ui"
+)
+
+var generateClientAPIKeyCmd = &cobra.Command{
```

Comment:
Can we change **terminology** to ClientToken?
![image](https://github.com/groundcover-com/cli/assets/11520905/54dc26c2-6448-4335-808a-7d75925b4f13)